### PR TITLE
Add thing name to expanded UI.

### LINKED ISSUE #513: Expanded Thing UI - Thing Name
Show the name of a Thing in the Expanded Thing UI, as per the mockups. Otherwise there's no way to tell what thing you're controlling from its expanded view.

### PR DESCRIPTION
**Pull request by mrstegeman**
In addition:
* Fix some scrolling issues.
* De-duplicate code for simple thing views.
* Small bug fixes.

Fixes #513

### CODE REVIEW

File: static/css/thing.css [hunk 1]
```diff
@@ -179,3 +200,16 @@
   font-weight: bold;
 }
 
+.thing-title {
+  width: 100%;
+  text-align: center;
+  position: fixed;
+  top: 20px;
```
Comment (hobinjk) [inline static/css/thing.css]: Should be in rem

File: static/css/thing.css [hunk 2]
```diff
@@ -179,3 +200,16 @@
...
+}
+
+.thing-title-icon {
+  height: 40px;
```
Comment (hobinjk) [inline static/css/thing.css]: These should all be in rem, I also think that 40px is slightly off for vertical alignment. For example, 
screen shot 2018-01-19 at 6 59 11 pm

**Comment by hobinjk** [comment]
Same goes for the onOffSwitch

**Comment by benfrancis** [comment]
I'm a bit confused by this diff because not all of the code seems related to this issue, but I agree with @hobinjk, why do you need to add extra HTML to the view of every thing type? Why can't there be one thing-name element in the ThingsScreen, whose value is set in showThing(), with an icon URL set based on a CSS class?

**Comment by hobinjk** [comment]
I agree that adding the element to ThingsScreen is a much nicer solution

**Comment by mrstegeman** [comment]
@benfrancis It was Friday and my brain was overcomplicating things.

**Pull request merged**